chat-staging.html

```diff
@@ -32,30 +32,52 @@
                 submitLabel: "Let's chat!"
             }
         };
+        const defaultLanguage = 'en-US';
+        const helplineLanguage = 'es';
+
         Twilio.FlexWebChat.createWebChat(appConfig).then(webchat => {
             const { manager } = webchat;
-            Twilio.FlexWebChat.MessagingCanvas.defaultProps.predefinedMessage.body = "How can I help?";
-        
-            //Posting question from preengagement form as users first chat message
-             Twilio.FlexWebChat.Actions.on("afterStartEngagement", (payload) => {
-                 const { question } = payload.formData;
-                 if (!question)
-                    return;
+            const serverlessBaseUrl = manager.serviceConfiguration.attributes.serverless_base_url;
```

Comment:
I don't think that `manager` here has a `serviceConfiguration` property.  When I add a line above this to `console.log(manager)`, that property doesn't appear.